elements/pfe-sass/functions/_custom-properties.scss

```diff
@@ -0,0 +1,134 @@
+///         (FYI, maps references: /elements/pfe-sass/src/variables/_colors.scss)
+///
+///     2. use like this:
+///         ```
+///         .my-element {
+///           color:       #{pfe-color(  surface--base--text  )};
+///           background:  #{pfe-color(  surface--base )};
+///         }
+///         ```
+
+@function pfe-color($cssvar, $category: 'color', $map: $pfe-colors) {
+    @if $category !="" {
+        $category: "#{$category}--";
+    }
+    // Start building the variable declaration
+    $var-declaration: "--#{$repo}-theme--#{$category}#{$cssvar}";
+    // If a fallback is defined, pass that in too
+    @if map-get($map, $cssvar) != null {
+        $var-declaration: "#{$var-declaration}, #{map-get($map, $cssvar)}";
+    }
+    // Return the variable declaration string
+    @return var(#{$var-declaration});
+}
+
+@function pfe-fallback($cssvar, $category: 'color', $map: $pfe-colors) {
+    @if $category !="" {
+        $category: "#{$category}--";
+    }
+    // Return the variable declaration string
+    @return #{map-get($map, $cssvar)};
+}
+
+/// Var
+///
+/// Returns CSS Var stack with exposed theme Var and respective SCSS fallback
+///
+/// @param {String} $cssvar - Variable name to be used and prepended with --pfe-theme
+/// @param {String} $category - Category name to be appended to variables within the map/system
+/// @param {Map} $map - SASS Map of variables
+/// @requires {String} $repo - Name of repo, which is "pfe"
+/// @use -
+///
+///     1.look up a color: /elements/pfe-sass/src/variables/_maps.scss
+///         (FYI, maps references: /elements/pfe-sass/src/variables/_colors.scss)
+///
+///     2. use like this:
+///         ```
+///         .my-element {
+///           padding:     #{pfe-var(  container-spacer  )};
+///           font-size:   #{pfe-var(  font-size  )};
+///         }
+///         ```
+
+@function pfe-var($cssvar, $category: '', $fallback: null, $map: $pfe-vars) {
+    $css-var-with-cat: $cssvar;
+
+    @if $category !="" {
+        $category: "#{$category}--";
+        $css-var-with-cat: "#{$category}#{$cssvar}";
+    }
+    // Start building the variable declaration
+    $var-declaration: "--#{$repo}-theme--#{$category}#{$cssvar}";
+
+    // If a fallback is defined, pass that in too
+    @if $fallback = null and map-get($map, $cssvar) != null {
+        @if $fallback == null {
+            $fallback: map-get($map, $css-var-with-cat);
+        }
+        $var-declaration: "#{$var-declaration}, #{$fallback}";
+    }
+    // Return the variable declaration string
+    @return var(#{$var-declaration});
+}
+
+/// Local
+///
+/// Returns CSS Var for the local component-scoped variable
+///
+/// @param {String} $cssvar - Variable identifiers which are postfixed and combined using '--'
+/// @param {String} $region - Identifies the region or slot to which this is assigned
+/// @requires {String} $repo - Name of repo, which is "pfe"
```

Comment:
Note the use of the `@requires` ahead of this comment.  That means that the global repo variable needs to exist.  If it has `@param` that would indicate it's an input parameter.  This should not be deleted because the function does use the global repo variable, thus it must be defined in the global scope and is required.